### PR TITLE
prototool: simplify setup to work cross-platform

### PR DESCRIPTION
**Pull request by ruseinov**
The setup with user ids didn't work for me as I'm on a mac with docker-machine. 

I have simplified it and it seems to work fine now. Protodocs error out though with 
```
lazyretina-tb:weave romanuseinov$ make protodocs
./scripts/build_protodocs_docker.sh
cmd/cleanproto/tests/codec.proto:20:9: "coin.Coin.whole" is already defined in file "coin/codec.proto".
cmd/cleanproto/tests/codec.proto:23:9: "coin.Coin.fractional" is already defined in file "coin/codec.proto".
cmd/cleanproto/tests/codec.proto:26:10: "coin.Coin.ticker" is already defined in file "coin/codec.proto".
cmd/cleanproto/tests/codec.proto:17:9: "coin.Coin" is already defined in file "coin/codec.proto".
cmd/cleanproto/tests/codec.proto:41:3: "coin.Coin" seems to be defined in "coin/codec.proto", which is not imported by "cmd/cleanproto/tests/codec.proto".  To use it here, please add the necessary import.
make: *** [protodocs] Error 1
```

### CODE REVIEW

**Comment by husio** [comment]
It does not work on linux. I am getting
```
$ make protoc
docker run --rm -v /home/piotr/weave:/work iov1/prototool:v0.2.0 prototool format -w
./scripts/build_protodocs_docker.sh
cmd/cleanproto/tests/codec.proto:20:9: "coin.Coin.whole" is already defined in file "coin/codec.proto".
cmd/cleanproto/tests/codec.proto:23:9: "coin.Coin.fractional" is already defined in file "coin/codec.proto".
cmd/cleanproto/tests/codec.proto:26:10: "coin.Coin.ticker" is already defined in file "coin/codec.proto".
cmd/cleanproto/tests/codec.proto:17:9: "coin.Coin" is already defined in file "coin/codec.proto".
cmd/cleanproto/tests/codec.proto:41:3: "coin.Coin" seems to be defined in "coin/codec.proto", which is not imported by "cmd/cleanproto/tests/codec.proto".  To use it here, please add the necessary import.
make: *** [Makefile:68: protodocs] Error 1

```

File: Makefile [hunk 1]
```diff
@@ -11,12 +11,12 @@ MODE ?= set
 
 # for dockerized prototool
 USER := $(shell id -u):$(shell id -g)
-DOCKER_BASE := docker run --rm --user $(USER) --mount type=bind,source="$(shell pwd)",target=/work --tmpfs /tmp:exec iov1/prototool:v0.2.0
+DOCKER_BASE := docker run --rm -v $(shell pwd):/work iov1/prototool:v0.2.0
```
Comment (ethanfrey) [inline Makefile]: Nice simplification. I copied the old line.

If you don't reference the USER variable anywhere, you can remove that line

File: Makefile [hunk 2]
```diff
@@ -11,12 +11,12 @@ MODE ?= set
...
 PROTOTOOL := $(DOCKER_BASE) prototool
 PROTOC := $(DOCKER_BASE) protoc
 
 
-all: deps test
+all: test
```
Comment (ethanfrey) [inline Makefile]: :+1:

File: cmd/cleanproto/tests/test.sh [hunk 3]
```diff
@@ -6,6 +6,6 @@ cd $(dirname $(dirname "$0"))
 
 make build
 dest=`tempfile`
-./cleanproto < tests/codec.proto > $dest
+./cleanproto < tests/codec.proto.origin > $dest
```
Comment (ethanfrey) [inline cmd/cleanproto/tests/test.sh]: Nice rename so we don't pick this up

File: prototool.yaml [hunk 4]
```diff
@@ -9,6 +9,9 @@ protoc:
     - .
     - /usr/include
 
+excludes:
```
Comment (ethanfrey) [inline prototool.yaml]: Never realized it would look there.

File: scripts/build_protodocs_docker.sh [hunk 5]
```diff
@@ -6,8 +6,8 @@ set -o errexit -o nounset -o pipefail
 # so it doesn't need the pre-clean step
 
 user=$(id -u):$(id -g)
-protoc="docker run --rm --user ${user} --mount type=bind,source=$(pwd),target=/work --tmpfs /tmp:exec iov1/prototool-docker protoc"
-prototool="docker run --rm --user ${user} --mount type=bind,source=$(pwd),target=/work --tmpfs /tmp:exec iov1/prototool-docker prototool"
+protoc="docker run --rm -v $(pwd):/work iov1/prototool:v0.2.0 protoc"
```
Comment (ethanfrey) [inline scripts/build_protodocs_docker.sh]: Thank you.
I had this built local earlier... no wonder tests didn't fail for me here

**Pull request merged**